06_modules_lab/01.py

```diff
@@ -1,14 +1,4 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times.
-
-For example if program was started as:
-    python 01.py 5
-
-It should print:
-    Hello Python
-    Hello Python
-    Hello Python
-    Hello Python
-    Hello Python
-"""
+import sys
+num = sys.argv [1]
```

Comment:
Personally for a variable named `num` I'd convert to `int` here when reading from argv - `num = int(sys.argv[1])`